webtau-docs/webtau/REST/HTTP-calls.md

```diff
@@ -0,0 +1,75 @@
+# Overloaded Calls
+
+`http.get|post|put|delete` methods have overloads to let you supply additional data:
+* Query parameters
+* Request header
+* Payload (*where applicable*) 
+* Validation block
+
+Overloads maintain relative order, but you can omit any data.
+
+`http.get` quick example:
+
+```tabs 
+Groovy: 
+:include-groovy: com/twosigma/webtau/http/HttpGroovyOverloadsTest.groovy {entry: "get no query return syntax example", bodyOnly: true, title: "without query"}
+
+Java: 
+:include-java: com/twosigma/webtau/http/HttpJavaOverloadsTest.java {entry: "getNoQueryReturnSyntaxExample", bodyOnly: true, title: "without query"}
+``` 
+
+# Implicit statusCode Check 
+
+If your code omits validation block, or if validation block doesn't assert `statusCode` an implicit status code validation
+be performed. 
+
+| Method             | Implicitly Expected Code |
+| ------------------ |--------------------------|
+| GET                | 200                      |
+| POST               | 201                      |
+| PUT                | 200                      |
+| PUT (no content)   | 204                      |
```

Comment:
Maybe clarify with `no response content`?  For delete this isn't particularly ambiguous given that deletes don't have requests but for put this is arguably slightly ambiguous.

Reply:
makes sense